### DISCUSSION TITLE
Concealed text highlighting

### DISCUSSION THREAD

**Discussion by LamprosPitsillos** [nvim-treesitter/nvim-treesitter#5864] [Q&A]
```scheme
;https://github.com/mushfiq814/dotfiles/tree/master/neovim/.config/nvim/after/queries
    ; list markers/bullet points
    (
      ([
        (list_marker_star)
        (list_marker_plus)
        (list_marker_minus)
      ]) @markdown_list_marker
      (#offset! @markdown_list_marker 0 0 0 -1)
      (#set! conceal "•")
    )
    
    ; checkboxes
    ((task_list_marker_unchecked) @markdown_check_undone (#set! conceal ""))
    ((task_list_marker_checked) @markdown_check_done (#set! conceal ""))
```
![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/61395246/0c6631c7-ff46-4685-acb8-9df0de09e6e6)

The text is highlighted with `@conceal` ,is there a way to specify better highlighting ? Or have it keep the original highlighting

**Comment by clason** [comment]
Your query does not contain `@conceal`, so that highlight group is not used. Something must be off. Please give complete, minimal reproducing steps with `nvim --clean` so we can verify.

**Reply by LamprosPitsillos** [reply]
Could i get some help with that ?
I tried 

`nvim --clean mymarkdown.md`
```
:lua vim.treesitter.start() 
:lua vim.treesitter.query.set("markdown","highlights","((task_list_marker_unchecked) @markdown_check_undone (#set! conceal \"\"))")  `
 ```
 But nothing changes
 
 Running `Inspect` 
 on normal conf:
![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/61395246/af257caf-496a-42f0-806e-3c5b01e94ffb)

on normal conf without conceal query
![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/61395246/5431a543-ec66-4703-9ef3-8174571f1abe)

 on `clean`
![image](https://github.com/nvim-treesitter/nvim-treesitter/assets/61395246/19912908-7c22-46c5-9099-3f58264a81cc)

So from what i understand it is trying to find highlight `@markdown_check_undone.markdown` but it doesnt exist.

**Reply by clason** [reply]
Well, then create it.

**Reply by LamprosPitsillos** [reply]
Is there a way to link it back to the original?

**Reply by clason** [reply]
Same as with any highlight group `:h :highlight-link`.
